specification/trace/semantic_conventions/http.md

```diff
@@ -153,6 +153,13 @@ This span type represents an outbound HTTP request. There are two ways this can
 
 For an HTTP client span, `SpanKind` MUST be `Client`.
 
+HTTP client spans SHOULD end sometime after all the response headers are received. This MAY or MAY NOT include time spent reading the response body.
```

Comment:
I think it's fine to have this as long as we use "SHOULD" instead of "MUST".

There can be scenarios where headers are not yet received but connection got reset, or timeout happened while reading headers - I don't feel we should document all of these.

Reply:
good point, I tried to make the language more precise, but ended up removing this line, I think(?) the `Note` stands ok on its own